tasks/check_for_ca.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+
+# Puppet Task Name: check_for_ca
+#
+# Explected possible inputs:
+#   $PT_master_replica_host
+#   $PT_puppetdb_database_host
+#   $PT_puppetdb_database_replica_host
+#   $PT_compiler_hosts
+#
+
+# A function to check for and remove signed certs for the infra
+function check_for_cert()
+{
+  cert="/etc/puppetlabs/puppet/ssl/ca/signed/${1}.pem"
```

Comment:
It is assumed that the certs are kept in this location.  However, the user could have configured it differently.  Consider using `cert=$(puppet config print cadir)/signed/${1}.pem`

Reply:
Also I would declare the ca directory globally so we don't have to constantly call puppet config print.

Then do `cert=$ca_dir/signed/${1}.pem`